packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js

```diff
@@ -32,7 +32,15 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
       })
       .$promise.then((repayments) => {
         this.allRepayments = repayments;
-        this.repayments = repayments.slice(0, LIMIT);
+        this.repayments = repayments
+          .slice(0, LIMIT)
+          .sort(({ creationDatetime: dateA }, { creationDatetime: dateB }) => {
+            if (new Date(dateA) < new Date(dateB)) {
```

Comment:
are you sure this works? when I run new Date() == new Date() or new Date() > new Date() or new Date() < new Date(), I always get false.

Reply:
Yes it is working, as explained to @frenautvh , the format 'YYYY-MM-DDTHH:MM:SS' make it working fine. Certainly date comparison is relying on this format

Reply:
new Date('2020-12-24T10:30:00') === new Date('2020-12-24T10:30:00')
The above statement returns false which is not true.

Reply:
You're right, and here's why:

`new Date() === new Date()` won't work as `new Date()` creates object. Object comparison, even if they have the same values, will always be falsy.

But instead, Date comparison with `<|>|<=|>=` (also with this format of date), will work, because they use .valueOf() under the hood.